Documentation/batch-scripts/mentoring.py

```
@@ -0,0 +1,436 @@
+            categories, medium, created_by, updated_by
+        ) = row
+
+        platform = None
+        if meeting_info:
+            try:
+                if isinstance(meeting_info, dict):
+                    platform = meeting_info.get("platform")
+                else:
+                    meeting_data = pyjson.loads(meeting_info)
+                    platform = meeting_data.get("platform")
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)